New partners page (#12564)
pulumi/docs: content/partners/_index.md

---
title: Pulumi Partners
layout: partners

> meta_title: Pulumi Partners - Accelerate Your Cloud Journey

Reviewer: ```suggestion
meta_title: Pulumi Partners - Accelerate Your Customers' Cloud Journey
```

> meta_desc: Explore Pulumi Professional Services for expert guidance on cloud infrastructure, automation, and best practices to streamline your cloud journey.

Reviewer: ```suggestion
meta_desc: Discover how to accelerate your cloud initiatives with Pulumi Partners. Whether you're looking to automate cloud capabilities our need expert guidance on infrastructure, automation, and best practices, our partners can help.
```

overview:

>     title: Accelerate Your Cloud Journey

Reviewer: ```suggestion
    title: Automate and Scale Cloud Projects
```

    description: |

>         Our mission is to empower our partners with the tools and resources they need to build robust, cloud-native infrastructure solutions. Comprehensive guides, detailed documentation, an engaged community, and expert support make partnering with us a seamless and productive experience. With our infrastructure-as-code platform and extensive API suite, you can create customized, scalable, and secure infrastructure solutions for your clients.

Reviewer: ```suggestion
        Pulumi partners with the worlds most popular cloud and SaaS companies and leading systems integrators to enable engineers and DevOps teams to deploy and manage cloud infrastructure at global scale.
```

    form:
        title: Contact our Partner Team

>         description: Ready to enhance your customers' cloud infrastructure with powerful automation and infrastructure-as-code capabilities? Drop our partner team a note to get started with Pulumi.

Reviewer: ```suggestion
        description: Are your customers demanding powerful automation through infrastructure as code and centralized environments, secrets and configuration? Connect with our partner team to learn more about working with Pulumi.
```

        form_id: d8eaf931-a94c-43cc-92d3-57557c6babf2
        campaign_id: 701PQ00000DrAWQYA3

types:
    title: Partnership Types
    items:
        - title: Cloud Partners
          icon:
            name: cloud
            color: yellow
          description: World’s leading hyperscalers that provide world class service to our mutual customers
        - title: Technology Partners
          icon:
            name: nodes
            color: salmon

>           description: Independent vendors integrate and test their products with Pulumi, enabling comprehensive, end-to-end cloud infrastructure solutions

Reviewer: ```suggestion
          description: Independent vendors integrate their products with Pulumi, enabling comprehensive, end-to-end cloud infrastructure solutions
```

        - title: System Integrators
          icon:
            name: shield
            color: purple
          description: Trusted and experienced advisors that provide customized solutions to clients implementing modern Infrastructure as Code

> 

Reviewer: Below the list of partners, include a link for "See our ecosystem partners" and link to the Pulumi Registry

case_studies:
    title: Case Studies
    items:
        - name: SST
          link: /case-studies/sst/
          logo: sst
          description: |
            SST builds a new version of SST (Ion) on Pulumi instead of CDK because of its limitations around speed, error handling, leaky abstractions, non-AWS provider support, and more.

>         - name: Fauna

Reviewer: Replace with Snowflake case study
